golem-component-compilation-service/docker/Dockerfile

```diff
@@ -16,8 +16,8 @@ WORKDIR /app
 COPY /target/$RUST_TARGET/release/golem-component-compilation-service ./
 COPY /golem-component-compilation-service/config/ ./config/
 
-RUN apt-get update && apt-get install -y libssl-dev
-RUN apt-get update && apt-get install -y ca-certificates
+RUN apt-get update --allow-releaseinfo-change && apt-get install -y libssl-dev
```

Comment:
not sure if  `--allow-releaseinfo-change` is needed here. Could be be needed if we used pinned version of ubuntu, but as we use ubuntu:latest it will handle the repo information change like suite, codename, version etc

Reply:
I think I needed `--allow-releaseinfo-change` to get something working.

Reply:
Just verified with fresh ubuntu vm that `--allow-releaseinfo-change` was not needed, also doing `apt-get update` twice is not needed, all of this can be replaced with
```
RUN apt-get update && apt-get install -y libssl-dev ca-certificates
```